main.go

```diff
@@ -42,9 +42,10 @@ func main() {
 	interval := flag.Int("interval", 1200, "Refresh interval in seconds")
 	profile := flag.String("profile", "", "The AWS Account profile")
 	secretName := flag.String("secretName", "ecr-auth-cfg", "The name of the secret")
+	secretType := flag.String("secretType", "Opaque", "The secret type to store")
 	flag.Parse()
```

Comment:
Please also do type assertion or validation after parsing the flags.

Reply:
will do